cmd/nerdctl/volume_create.go

```diff
@@ -17,6 +17,9 @@
 package main
 
 import (
+	"fmt"
+
+	"github.com/containerd/containerd/errdefs"
```

Comment:
Yes, but probably it should be a separate PR

Reply:
@djdongjin I do not think this will work, unless we update everything (including dependencies) to v2, as both packages will yield different results wrt `errors.Is`

Reply:
Correction: may actually work with 1.7.19 - still, certainly better as a separate PR.